### PR TITLE
task-4364: moved TransformTestExtension Util methods to TestPackutils

### PR DESCRIPTION
**Pull request by Chuks-Ajeh**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -479,6 +479,8 @@
                 </executions>
                 <configuration>
                     <release>${maven.compiler.release}</release>
+                    <source>10</source>
```
Comment (hugohills-regnosys) [inline pom.xml:482]: this seems wrong

File: src/main/java/com/regnosys/rosetta/common/transform/TestPackUtils.java [hunk 2]
```diff
@@ -20,4 +45,58 @@ public static PipelineModel createPipeline(TransformType transformType, String f
...
+
+    public static <T> T readFile(URL u, ObjectMapper mapper, Class<T> clazz) {
+        try {
+            return mapper.readValue(UrlUtils.openURL(u), clazz);
+        } catch (Exception e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    // TODO move to util class?
```
Comment (hugohills-regnosys) [inline src/main/java/com/regnosys/rosetta/common/transform/TestPackUtils.java:68]: remove comments and make methods public

Comment (Chuks-Ajeh) [reply src/main/java/com/regnosys/rosetta/common/transform/TestPackUtils.java:68]: done

File: pom.xml (hunk 1 above)
Comment (Chuks-Ajeh) [reply pom.xml]: removed

**Review by hugohills-regnosys — DISMISSED** [review]

**Review by hugohills-regnosys — APPROVED** [review]

**Review by PayalKhanna — APPROVED** [review]

**Pull request merged**